repository/migrations/sqlite/2021-09-17T10:00_create_requisition_table/up.sql

```diff
@@ -0,0 +1,20 @@
+-- Create requisition table.
+
+CREATE TABLE requisition (
+    id TEXT NOT NULL PRIMARY KEY,
+    requisition_number BIGINT NOT NULL,
+    store_id TEXT NOT NULL REFERENCES name(id),
```

Comment:
Looks like name and store might be the wrong way around in this sql?

```suggestion
    store_id TEXT NOT NULL REFERENCES store(id),
    name_id TEXT NOT NULL REFERENCES name(id),
```

Reply:
Thanks James ! Nicely spotted